src/jshint.js

```diff
@@ -423,7 +423,8 @@ var JSHINT = (function() {
 
     removeIgnoredMessages();
 
-    if (JSHINT.errors.length >= state.option.maxerr)
+    var errors = JSHINT.errors.filter(function(e) { return /E\d{3}/.test(e.code); });
+    if (errors.length >= state.option.maxerr)
```

Comment:
Mind adding braces around the IfStatement body?

Reply:
Yep. Done. I actually did read up your coding style in contribution guidelines but missed this one out because the previous if statement (before this commit) had no braces. My bad.